pkgs/top-level/all-packages.nix

```diff
@@ -35371,4 +35371,6 @@ with pkgs;
   honeyvent = callPackage ../servers/tracing/honeycomb/honeyvent { };
 
   mictray = callPackage ../tools/audio/mictray { };
+
+  curtail = callPackage ../applications/graphics/curtail { };
```

Comment:
Please sort this into a fitting category and sort there alphabetically.

Reply:
Done